nixos/modules/services/development/zammad.nix

```diff
@@ -211,6 +242,10 @@ in
         assertion = cfg.database.createLocally -> cfg.database.passwordFile == null;
         message = "a password cannot be specified if services.zammad.database.createLocally is set to true";
       }
+      {
+        assertion = cfg.redis.createLocally -> cfg.redis.host == "localhost";
+        message = "the redis host must be localhost if services.zammad.redis.createLocally is set to true";
+      }
```

Comment:
Why not use the redis socket for local things?

Reply:
Because Zammad wants to use TCP for redis by default, and the module just adopts this.